context.go

```diff
@@ -0,0 +1,25 @@
+package logf
+
+import "context"
+
+type contextKey int
+
+const (
+	contextKeyLogger = contextKey(iota)
+)
+
+// ContextWithLogger returns new context with the provided Logger inside it.
+func ContextWithLogger(parent context.Context, logger *Logger) context.Context {
+	return context.WithValue(parent, contextKeyLogger, logger)
+}
+
+// LoggerFromContext returns Logger from context previously added to it with ContextWithLogger
+// or nil if there is no Logger in context.
+func LoggerFromContext(ctx context.Context) *Logger {
```

Comment:
Don't you think that logger is the only type to be wrapped into context in the whole package?
IMHO 'Logger' prefix is not needed.